fetch-in-react/src/app.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import UserList from './user-list';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      users: [],
+      isLoading: true
+    };
+  }
+
+  componentDidMount() {
+    /* your code here */
+    this.setState({ isLoading: false });
+    fetch('https://jsonplaceholder.typicode.com/users')
+      .then(res => {
+        return res.json();
+      })
+      .then(users => {
+        this.setState({ users });
```

Comment:
Here is where you want to set `isLoading` to `false`. You can set both the `users` property and `isLoading` property in the same `setState` call.
```JAVASCRIPT
this.setState({ users, isLoading: false });
```